.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,27 @@
+# Type of Issue:
+Some example types are:
+* bug in science or answers
+* bug in software
+* science enhancement (changes answers and requires science development)
+* software enhancement
+* FATES change
+* Discussion
+
+
+## Summary of Issue:
+
+## Details of Issue:
+(Be sure to give enough details that the issue can understood sufficiently to replicate
+ or to make the required changes)
+
+# For bugs be sure to fill out the following (otherwise remove):
+-----------------------------------------------------------------
+# Expected behavior and actual behavior:
+
+# Steps to reproduce the problem:
```

Comment:
Steps to reproduce the problem can be useful in some cases. However, I'd prefer if this and the machine (if relevant... usually it isn't) were folded into the "details" section. You already pretty much ask for this in the parenthetical in that section... we should just make it clear that they should give enough information to reproduce the issue there, and not ask for it again separately.